packages/@aws-cdk/aws-kinesisfirehose/README.md

```diff
@@ -311,6 +311,62 @@ new DeliveryStream(this, 'Delivery Stream', {
 });
 ```
 
+## Data Processing/Transformation
+
+Data can be transformed before being delivered to destinations. There are two types of
+data processing for delivery streams: record transformation with AWS Lambda, and record
+format conversion using a schema stored in an AWS Glue table. If both types of data
+processing are configured, then the Lambda transformation is performed first. By default,
+no data processing occurs. This construct library currently only support data
+transformation with AWS Lambda. See [#15501](https://github.com/aws/aws-cdk/issues/15501)
+to track the status of adding support for record format conversion.
+
+### Data transformation with AWS Lambda
+
+To transform the data, Kinesis Data Firehose will call a Lambda function that you provide
+and deliver the data returned in lieu of the source record. The function must return a
```

Comment:
I definitely used this word originally but unnecessarily flowery
```suggestion
and deliver the data returned in place of the source record. The function must return a
```

Reply:
lol +1 :)